project-1/requirements.txt

```diff
@@ -0,0 +1,4 @@
+selenium==4.14.0
+gspread==5.11.3
+pandas @ file:///C:/ci/pandas_1650373458095/work
```

Comment:
The new format you used is now proper, however, `file:///C:/ci/pandas_1650373458095/work` is only available on your computer. Use the version number instead or a link that's publicly available to anyone willing to replicate your project.